Header header

uint8 elec_level
uint8 PC1_level
uint8 motorH_level
uint8 motorL_level

uint8 charger_status
